recipes/zoom/webview.js

```diff
@@ -33,8 +33,8 @@ module.exports = (Ferdium, settings) => {
 
     if (link || button) {
       const url = link ? link.getAttribute('href') : button.getAttribute('title');
-      
-      if (!Ferdium.isImage(url)) {
+
+      if (!Ferdium.isImage(link)) {
```

Comment:
What happens if the user clicked on a button (and the `link` variable is undefined)?

Reply:
Good question. Maybe this should only be checked if it is NOT a button?

Reply:
Inside Ferdium.IsImage we make a check and return false if "link" is undefined